src/main/java/org/fountainmc/forge/Instance.java

```diff
@@ -0,0 +1,11 @@
+package org.fountainmc.forge;
+
+import org.fountainmc.api.command.CommandManager;
+import org.fountainmc.api.plugin.PluginManager;
+
+public class Instance {
```

Comment:
Instead of having these in their own class, put them in MixinMinecraftServer and initialize them in the constructor.